xarray/test/test_quantities.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
```

Comment:
I see. I was rather thinking of this test-file as specialised towards physical units handling with quantities (i.e. as a regression-test database geared towards #525). I'd rather make a separate test-case for general subclasses, maybe with a dummy subclass.